CorsixTH/Lua/hospital.lua

```diff
@@ -1232,7 +1234,7 @@ end
  appropriate epidemic if so.
  @param patient (Patient) patient to determine if contagious]]
 function Hospital:determineIfContagious(patient)
-  if patient.is_emergency or not patient.disease.contagious then
+  if patient.is_emergency or not patient.disease.contagious or self.epidemics_off then
```

Comment:
I'd write your addition as first condition rather than as last condition, perhaps even as separate `if` (but that may be just me).

The new condition is a big "everything epidemics doesn't apply" switch. That is at a higher abstraction level than digging around in specific patient types and diseases to decide whether the patient may be contagious.

When reading the code the mind has to make abstraction level switches from details to high level and back to some other details. By doing high level conditions first, you ease the reader from higher level concerns to lower level concerns.   If you do this consistently, it also becomes a pattern. Higher level concerns are near the top of a function. That makes finding and checking easier, eventually.

The name of the variable is quite horrible (it has negative logic which gets very complicated very quickly in code), but that's not your fault and out of scope in this patch.